docs/api/screen.md

```diff
@@ -142,6 +144,8 @@ Returns [`Point`](structures/point.md)
 Converts a screen DIP point to a screen physical point.
 The DPI scale is performed relative to the display containing the DIP point.
 
+Not currently supported on Wayland.
```

Comment:
It looks like the implementation technically just returns the original Point with the point values Floored. I wonder if we should maybe be more explicit about what we do on Wayland, but also document that it might change in the future (if we add better Wayland support)